### PR TITLE
Test 79 As a user I should be able to download generated PTO document

### PR DESCRIPTION
**Pull request by kristiyanplpeev**
Front-end added functionality for generating and downloading a PDF file.

### CODE REVIEW

File: client/src/components/PTODetails/PTODetails.tsx [hunk 1]
```diff
@@ -110,6 +116,10 @@ class PTODetails extends Component<PTODetailsProps, PTODetailsState> {
       </div>
     );
   }
+
+  private calculateWorkingDays(daysStatuses: HolidayDaysInfoType): number {
+    return daysStatuses.filter((el) => el.status === "workday").length;
```
Comment (MilenIvanovv) [inline client/src/components/PTODetails/PTODetails.tsx:121]: Use the status "working" from enum

**Review by MilenIvanovv — CHANGES_REQUESTED** [review]
LGTM left one comment for a minor change

File: client/src/components/PTODetails/PTODetails.tsx (hunk 1 above)
Comment (kristiyanplpeev) [reply client/src/components/PTODetails/PTODetails.tsx:121]: done.

**Review by MilenIvanovv — APPROVED** [review]
LGTM

**Pull request merged**